.github/release-drafter.yml

```diff
@@ -0,0 +1,33 @@
+categories:
+  - title: 'Features'
+    labels:
+      - 'enhancement'
+  - title: 'Bug Fixes'
+    labels:
+      - 'bug'
+  - title: 'Documentation'
+    label: 'documentation'
+  - title: 'Maintenance'
+    labels:
+      - 'chore'
+      - 'deployment'
+      - 'translation'
+  - title: 'Dependencies'
```

Comment:
What do you think about adding [`collapse-after`](https://github.com/release-drafter/release-drafter#categorize-pull-requests) for this category?  We'll probably have a lot of these thanks to dependabot

Reply:
That's a good idea